lib/Request.js

```diff
@@ -34,7 +34,7 @@ class Request extends CallableInstance {
   on (...args) {
     // should we warn/block users when they try
     // to listen to an event that doesn't exist?
-    this._emitter.on(...args)
+    this._remit._emitter.on(...args)
```

Comment:
So these events that are made by clients will be for `data`, `sent` etc for _this request_, so this bit should still be using `this._emitter` rather than the `this._remit` one.